src/test/java/io/nats/client/impl/HeadersTests.java

```diff
@@ -164,6 +163,7 @@ private void put(
     private void assertKeyIgnoreCaseContainsValues(Headers headers, List<String> keys, List<String> values) {
         for (String k : keys) {
             List<String> hVals = headers.getIgnoreCase(k);
+            assertNotNull(hVals);
```

Comment:
pretty much all of the test changes where to check for null as the new annotation shows up with warnings from IntelliJ